src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories_catalog.html.twig

```diff
@@ -25,22 +25,21 @@
 <div class="ps-dropdown dropdown btn-group bordered mb-1">
   {% for menu in topMenuData %}
     {% set refMenu = menu.refMenu %}
-    <div id="{{refMenu}}" class="dropdown-label" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+    <div id="{{refMenu}}" class="dropdown-label " data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
```suggestion
    <div id="{{refMenu}}" class="dropdown-label" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

or was there a reason for adding a space ?

Reply:
No reason at all :D :D Just forgetted